src/ParticleDataLoader.h

```diff
@@ -50,6 +50,29 @@ class LoadHOOMDGroupPositionVelocityMass
     const unsigned int* const m_indexes;
     };
 
+//! Load HOOMD particle in a group from an index
+class LoadHOOMDGroupVelocityMass
```

Comment:
Do you want to call it "Particle" instead of "HOOMD"? Or "MDParticle"? It doesn't make a difference, it is just less immediately obvious what it is (especially if people don't look at the MPCD counterpart)... optional suggestion. feel free to ignore.